rules.adoc

```diff
@@ -150,15 +150,14 @@ Within a limited time, the team is asked to perform as many passes as possible.
 Teams are free in how they perform the passes, as long as the following constraints are met:
 
 - A valid pass must be at least *1.5m* long.
-- The shooter of the previous pass (valid or invalid) can not be the receiver of the current pass (this means you need *at least 3 robots*).
-- A pass must be directed at least *10°* away from where the last pass (valid or invalid) came from (so two robots can not pass back and forth without changing positions).
+- The shooter of the previous pass can not be the receiver of the current pass, where the last pass does not depend on whether it is valid or not. This means that two robots can not pass back and forth without changing positions. You need *at least 3 robots*.
```

Comment:
I don’t understand well what you explained.
What is the difference between ”The shooter of the previous pass can not be the receiver of current pass” and ”two robots  can not pass back and forth without changing position”?

Reply:
"The shooter of the previous pass can not be the receiver of current pass" means that if robot A passes to robot B, than robot B can not pass to robot A, regardless of the positions. So it has to pass to robot C. That's where the requirement for 3 robots comes from.

"two robots can not pass back and forth without changing position” is related to the 10° angle: If robot A and robot B stand next to each other (close) and another robot C stands somewhere else; If robot A passes to robot C, then robot C can not pass to either robot, because the pass angle is smaller than 10°. Robot B must move away from robot A so that the pass angle between the pass A -> C and C -> B is larger than 10°.

We might need some figures to better demonstrate this :thinking:

Reply:
![rule_sample](https://user-images.githubusercontent.com/68051317/114552491-9f395580-9c9f-11eb-9945-8228eebd704e.png)

The intention that you add the part ("two robots can not pass back and forth without changing positions") is to avoid a situation which is described in the attached picture (The situation follows Line 153 rule while conflicting Line 154).
Is that correct?

If so, I think the part does not describe the intention well, and therefore, we suggest removing it from the proposed Line 153.

Reply:
I've adapted the two lines in #2. Is this now better? Did I address your concern?

Reply:
Thank you for adapting, but what we (RoboDragons inc. Mayu and me) are arguing is the Line 161. The Line 161 was obscure, and we tried improving it by re-describe the content in the Line 153 and 154 (" , where the last pass does not depend on whether it is valid or not"). However, #2 does not include this improvement. So, if there is no concern about the improvement, could you add the content to #2 ?

Reply:
I see, so you guys were primary confused by the definition of valid and invalid passes?

In #2 I've tried to make the whole definition more formal. It now includes a definition of passes in general, and the constraints for valid passes for scoring.
Does it read better now? Or do we actually still have different understandings of the rules in the first place?